### ISSUE TITLE
Add new member on the picoterm repository ?

### ISSUE THREAD

**Issue by mchobby** [RC2014Z80/picoterm#12]
Hi,
I'm actively work on the PicoTerm and there is still a lot work to do there. I'm not a C guru but can provide useful help on this project.
It would be more efficient to work directly on the RC2014Z80/picoterm repository (instead of doing fork -> Sync fork -> Code feature on fork -> Pull-Request -> Wait acceptance --> Sync fork again -> Code again ..... ).
What do you thing? Would it be a good idea to add me as a developer on the repository.

Here what I do see as immediate tasks:

1. Porting Menu to 40 column version (having both version stable).
2. Moving more code to /common 
3. Code formatting and code cleaning (would be required to ease work)
5. Improving config Flash storage (Magic Key + Version) to append new parameters more easily.
6. Appending UART port configuration to Menu & store it into Flash storage.
7. Nice to have: Configure string for button A B C. Eg: sending CONSOLE 1, Popup the menu, display SD file to send (see below).

Some other ideas: 
8. Nice to have: include all keyboard layouts into to firmware and select it via the menu.
9. Nice to have: Adding Hex/Bin/Dec quick converter and calculation (that would be so useful for newbies using the monitor)
10. Nice to have: SD Card support --> Allowing to send HEX file from SD  
11. Nice to have: (wish) removing space in directory name in the repository

Cheers,
Dominique

**Comment by RFC2795** [comment]
Hi Dominique,

Yes, I would be happy to have you as a member on this repository :-)  I've sent an invite to you, so I hope this works.

My only request is that after any major updates that you build it for the different keyboard locales and put a copy of the uf2 in the Build directory.  I know that setting up the toolchain for this can be complicated for a lot of people, so being able to download the uf2 file and flash it makes things as easy as possible for them.

Cheers

Spencer

**Comment by mchobby** [comment]
Hi Spencer,
Thank for the invitation :-) 
I will publish the UF2 for each new version. I'm currently working on the version 1.1 code (so I placed a 1.1.1 in version number to distinguish beta code from final release code).
When code/feature would be stable, I will jump to 1.2 and publish the UF2 files together.

This evening I did make the necessary to include the menu + Flash config writing into the 40 cols version. Color selection & storage in flash doesn't concerns the 40 column version... but I would like the both terminal to share more common source code as soon as possible. 
This will allows us to make them evolving in parallel with version numbering.
